source/Octopus.Server.Client/Operations/RegisterMachineOperation.cs

```diff
@@ -65,16 +63,26 @@ public RegisterMachineOperation(IOctopusClientFactory clientFactory) : base(clie
         /// </exception>
         public override void Execute(IOctopusSpaceRepository repository)
```

Comment:
Side note: It would be great if we could remove these non-async methods. It's an annoying amount of duplication and I'm not sure if they are used anymore? They don't seem to be tested.

Reply:
Interesting. Might be worth asking more widely if these could be trashed